readme.md

```diff
@@ -73,15 +73,46 @@ The swagger files are compiled, and then run, which pings the mock server (to ve
 
 This document contains some additional [technical details](https://github.com/Azure/autorest.csharp/blob/feature/v3/test/README.md).
 
-### Testing generator changes against Azure SDK without a PR
-
-These instructions are only a general outline, see [the script](https://github.com/Azure/autorest.csharp/blob/feature/v3/eng/UpdateAzureSdkForNet.ps1) for details:
-
-- Use `dotnet pack` to package up a version of the generator
-- `dotnet pack -o directory` (where directory is replaced with the name of the directory)
-- Copy created nuget package to your local nuget source
-- Update package.props
-- `dotnet restore -S directory` (where directory is replaced with the name of the directory)
+### Validate generator changes against Azure SDK before merging your autorest.csharp PR
+
+When the automatic PR is created for azure-sdk-for-net if there are any issues found all other changes to autorest.csharp are blocked until those issues are resolved.  This means we should be certain that the changes we are making create the expected result in azure-sdk-for-net prior to merging our PR.  In time we should build this into the autorest.csharp CI such that it happens automatically, but in the meantime a few manual steps can help avoid any delays.
```

Comment:
As @AlexanderSher  is working on the CI improvement,  do we still need those steps after the CI work completes?

Reply:
I have no idea @AlexanderSher hasn't shared that with the broader community yet, however I do know it is needed now and after any of those changes get deployed we can always adjust this process then